customers/services/profile.py

```diff
@@ -127,6 +127,35 @@ def get_profile(self, id: UUID) -> HelsinkiProfileUser:
         user = self.parse_user(response.pop("profile"))
         return user
 
+    def get_my_profile(self) -> Optional[HelsinkiProfileUser]:
+        query = """
+            query MyProfile {{
+                myProfile {{
+                    id
+                    first_name: firstName
+                    last_name: lastName
+                    primary_email: primaryEmail {{
+                        email
+                    }}
+                    primary_phone: primaryPhone {{
+                        phone
+                    }}
+                    primary_address: primaryAddress {{
+                        address
+                        postal_code: postalCode
+                        city
+                    }}
+                }}
+            }}
+        """
+
+        response = self.query(query)
+        my_profile = response.pop("myProfile")
```

Comment:
We should do here a `response.get`, on the very unlikely case that the response didn't contain the `myProfile` prop, it could fail unexpectedly.